packages/app/src/specs/InlineRunAllSpecs.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <Tooltip
+    placement="right"
+    class="h-full truncate"
+    data-cy="tooltip"
+  >
+    <button>
+      <IconActionPlaySmall
+        size="16"
+        stroke-color="gray-700"
+        fill-color="transparent"
+        hocus-stroke-color="indigo-500"
+        hocus-fill-color="indigo-100"
+        class="inline-flex align-text-bottom"
+        data-cy="play-button"
+        @click.stop="emits('runAllSpecs')"
+      />
+    </button>
+    <template
+      #popper
+    >
+      <span
+        class="font-normal text-sm inline-flex"
+        data-cy="tooltip-content"
+      >
+        Run {{ specNumber }} specs
```

Comment:
We need to handle pluralization here, we could use the i18n for this.

<img width="180" alt="Screen Shot 2022-11-21 at 4 16 03 PM" src="https://user-images.githubusercontent.com/8340719/203159849-24909c42-c2ef-4acf-b125-b9208f7fcbc3.png">

Reply:
Oh it's already in the main Spec Explorer, we can do that here.
```suggestion
       {{ t('specPage.runAllSpecs', specNumber) }}
```

Reply:
done